nectar/CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   New command to archive swaps: `nectar archive-swap <swap id>`.
     The command should only be used while nectar is stopped as it updates the database.
     Archiving a swap stops nectar to resume its automated execution, `create-transaction` command can then be used to recover funds.
+-   Add an optional `fund_amount` parameter to the `create-transaction` command.
```

Comment:
Why adding this feature?

Reply:
I felt that it is actually quite important given that you may want to use these commands to recover funds. If you can't specify the amount, it is impossible to refund/redeem and over/underfunded HTLC.

It was brought to my attention again because I removed the amounts from the `build_refund` and `build_redeem` functions and those were used within those commands.